Healthy/Classes/Generated/Strings+Generated.swift

```diff
@@ -37,4 +37,4 @@ private final class BundleToken {
     #endif
   }()
 }
-// swiftlint:enable convenience_type
+// swiftlint:enable all
```

Comment:
Was this added to the SwiftGen template? Otherwise, it should be removed. 

T.B: We already ignore generated files from linting.